source/technical/networks/networks_stitching.rst

```diff
@@ -162,3 +166,24 @@ To obtain an account to access AL2S OESS portal, users should contact Internet2.
 - `AL2S FAQ <https://www.internet2.edu/products-services/advanced-networking/layer-2-services/#service-faq>`_
 - `Using OESS <https://docs.globalnoc.iu.edu/sdn/oess/using-oess.html#Multipoint%20Static%20MAC%20Addresses>`_
 
+
+Connecting Stitchable Isolated Networks across Chameleon Sites
+______________________________________________________________
+
+1. Create isolated networks by specifying the "exogeni" provider. Follow the documentation for :ref:`network-stitchable-create`
+   A "stitchable" VLAN tag will be returned and "Physical Network" will appear as "Exogeni" on the dashboard.
+   This step will be executed the same way on both UC and TACC sites. 
+
+2. After having stitchable isolated networks on UC and TACC sites, a request should be sent to the Help Desk for creation of AL2S circuits. 
```

Comment:
Please link to the [Help Desk ticket submission page](https://www.chameleoncloud.org/user/help/ticket/new/) here.